difflint/lib.py

```diff
@@ -36,24 +24,7 @@ def lint_list(file_list):
     Returns: A mapping of filenames to their linted
              output as a LintOutput object.
     '''
-    # Mapping of {filename : LintOutput}
-    lint_mapping = {}
-
-    for f in file_list:
-        root, ext = splitext(f)
-        if ext == ".js":
-            lint_output_bytes = None
-            warnings_present = False
-            try:
-                lint_output_bytes = check_output([LINT_PATH, f])
-            except CalledProcessError as e:
-                warnings_present = True
-                lint_output_bytes = e.output
-            lint_mapping[f] = LintOutput(lint_output_bytes.decode(),
-                                         warnings_present)
-        # TODO: Other file extensions/linters will be added here.
-
-    return lint_mapping
+    return {f: lint(f) for f in file_list}
```

Comment:
nit: Feels like there should be a space before `:`, but I'm not sure what Pylint would say.